TESTPLAN.md

```diff
@@ -1,207 +1,15 @@
-# Test Plan for the 57-Perception-Study Addon
+# Test Plan for The ESPER experiment - Firefox Pioneer Study
 
 ## Automated Testing
 
-`npm test` does **optimistic testing** of the *commonest path* though the study for a user
-
-- prove the notification bar ui opens
-- *clicking on the left-most button presented*.
-- verifying that sent Telemetry is correct.
+`npm test` verifies the telemetry payload as expected at startup with a clean profile.
 
 Code at [./test/functional_test.js].
 
 ## Manual / QA TEST Instructions
 
-Assumptions / Thoughts
-
-1.  Please ask if you want  more command-line tools to do this testing.
-
-### BEFORE EACH TEST: INSTALL THE ADDON to a CLEAN (NEW) PROFILE
-
-0.  (create profile:  https://developer.mozilla.org/en-US/Firefox/Multiple_profiles, or via some other method)
-1.  In your Firefox profile
-2.  `about:debugging` > `install temporary addon`
-
-As an alternative (command line) cli method:
-
-1. `git clone` the directory.
-2. `npm install` then `npm run firefox` from the Github (source) directory.
-
+See `README.md` for instructions on how to build, install and run this add-on.
```

Comment:
FWIW, we have a 'real' looking test plan at:  https://github.com/raymak/cfr-focused-release/blob/master/TESTPLAN.md  

In particular, we found the SETUP / ACTION / VERIFY framing to be interpretable.